packages/core/docs/.vuepress/config.js

```diff
@@ -107,7 +107,7 @@ module.exports = {
           collapsable: false,
           children: [
             ['/commercetools/', 'Introduction'],
-            ['/commercetools/getting-started', 'Getting started'],
```

Comment:
Maybe move it to How-to-guide? Also change the name to just sth like "Setting up store". Just thinking ;)

Reply:
I think it could be good idea to move it, but also mention in `Getting staered` section or in landing page as the help for new users to run their first project. Wdyt @lukaszjedrasik @filipsobol ?

Reply:
I moved it to `Getting started` section for now.